e2e-cypress/files/package.json

```diff
@@ -19,12 +19,13 @@
   },
   "private": true,
   "devDependencies": {
-    "@types/node": "^20.10.0",
-    "cypress": "^13.6.0",
+    "@types/node": "^22.4.1",
+    "cypress": "^13.13.1",
     "cypress-junit-reporter": "^1.3.1",
```

Comment:
I guess the `cypress-junit-reporter` can now be removed here.

Reply:
you are right